content/en/contribute/docs/style-guide.md

```diff
@@ -262,6 +230,19 @@ represents.
 | The doc in the database is a "record". | The doc in the database is a "record." |
 | The copy is called a "fork". | The copy is called a "fork." |
 
+### Number formatting
+Avoid the use of comma or period as thousands separator since it can be confused for a decimal point in some countries. Either use no separator for small numbers or a unicode _Thin Space_ which is often recommended for international documents[^digit-grouping].
+
+There's a shortcode `format-number` which will replace `_` with _Thin Space_ to simplify formatting in this way.
+
+| Do                                    | Don't   |
+|---------------------------------------|---------|
+| `{{</* format-number 10_000 */>}}`    | 10,000  |
+| `{{</* format-number 10_000 */>}}`    | 10.000  |
+| `{{</* format-number 1_000_000 */>}}` | 1000000 |
+
+[^digit-grouping]: https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping
```

Comment:
While I didn't know how to do citations, so far we haven't done them in our docs.  Not too big of a deal, but I think just a normal hyperlink on [line 234](https://github.com/medic/cht-docs/pull/1200/files#diff-b13c104ce4c4d34c0be07c5353173d6482a4f0b4651e437a639c03e5997abf44R234) is more inline with our existing styles.

Reply:
I was already on the fence about footnotes, I'll change it to be more like the rest of the docs